Gaijinizer patch/Map237.txt

```diff
@@ -56,26 +60,29 @@ They are more than food. It seems that you like
 \\{家の中にずっと缶詰じゃ……\\}ｽﾞｽﾞｽﾞ
 \\{食い物で魔力変換も必要ないだろうね\\}ﾑｼｬｱ
 > CONTEXT: Map237/events/1/pages/0/42/Dialogue/enemy19
-Hmm! !\\} Mog mog. \\{You can always stay in the house
-......\\} Zuzu's. \\{You won't need to enchant and
-convert magic\\} Mushar#MTLed
+Hmmph!!\\} *chew chew*
+\\{You're always stuck at home...\\} *crunch crunch*
+\\{There's no need to convert food into magic\\} *gulp*#CheckLater
 > END STRING
```

Comment:
Note

Reply:
More onomatopoeia uncertainty unless we're happy to localise as-is.

Reply:
I think this is fine.